tests/test_opam_file.ml

```diff
@@ -27,7 +31,8 @@ description: "This package is nice"
 |}
   in
   let expected_v1 =
-    {|opam-version: "2.0"
+    {|x-commit-hash: "6814f8b26946358c72b926706f210025f36619b0"
```

Comment:
I'm surprised it's printed as the first field!

Reply:
I looked this up, it does'nt look like we have much control upon this, we are just calling write_to_string

Reply:
Yeah I wasn't expecting we'd have control over this. It doesn't matter much anyway!